web_responsive/static/src/css/web_responsive.scss

```
@@ -16,6 +16,18 @@
     transform: none !important;
 }
 
+// Support for long titles
+@include media-breakpoint-up(md) {
+    .o_content, .modal-content {
+        .o_form_view {
+            .oe_title {
+                /* Button-box has a hardcoded width of 132px per button and have three columns */
+                width: calc(100% - 450px) !important;
```

I tested this and it seems to have the same effect, so as a good rule of thumb, always avoid `!important` clauses:

```suggestion
                width: calc(100% - 450px);
```